org.civicrm.bootstrapcivicrm/scss/bootstrap/bootstrap.scss

```diff
@@ -5,14 +5,14 @@
 @import "mixins/*";
 
 #{$bootstrap-namespace} {
-    @import "vendor/text-angular/text-angular";
-    @import "vendor/ui-bootstrap/ui-bootstrap";
-    @import "vendor/ui-select/ui-select";
+  @import "vendor/ui-bootstrap/ui-bootstrap";
+  @import "vendor/ui-select/ui-select";
 
-    @import "overrides/style/*";
-    @import "components/*";
+  @import "overrides/style/*";
+  @import "components/*";
 }
 
+@import "vendor/text-angular/text-angular";
```

Comment:
Because it was showing the `input` and this import has the style to hide it. The other pages that already use text-angular will continue working.

![image](https://cloud.githubusercontent.com/assets/1280255/17884165/11495f80-68ed-11e6-8f8d-f33b0b2f84d1.png)